staging/README.md

```diff
@@ -46,3 +46,53 @@ import (
 
 Once the change-over to external repositories is complete, these repositories
 will actually be vendored from `k8s.io/<package-name>`.
+
+## Creating a new repository in staging
+
+### Adding the staging repository in `kubernetes/kubernetes`:
+
+1. Send an email to the SIG Architecture
+[mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-architecture)
+and the mailing list of the SIG which would own the repo requesting approval
+for creating the staging repository.
+
+2. Once approval has been granted, create the new staging repository.
+
+3. Add a symlink to the staging repo in `vendor/k8s.io`.
+
+4. Update [`hack/import-restrictions.yaml`](https://github.com/kubernetes/kubernetes/blob/master/hack/import-restrictions.yaml)
+to add the list of other staging repos that this new repo can import.
+
+5. Add all mandatory template files mentioned in
+https://github.com/kubernetes/kubernetes-template-project.
+
+6. Make sure that the `.github/PULL_REQUEST_TEMPLATE.md` and
+`CONTRIBUTING.md` files mention that PRs are not directly accepted to the repo.
+
+### Creating the published repository
+
+1. Create an [issue](https://github.com/kubernetes/org/issues/new?template=repo-create.md)
+in the `kubernetes/org` repo to request creation of the respective published
+repository in the Kubernetes org. The published repositories need specific
+access rules and branch settings.
```

Comment:
The published repositories must have an initial empty commit.

Reply:
Added.